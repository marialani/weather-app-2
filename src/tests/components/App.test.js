import { render } from "@testing-library/react";
import React from "react";
import App from "../../components/App";

test("renders correctly", () => {
  const { asFragment } = render(<App />);

  expect(asFragment()).toMatchSnapshot();
});
