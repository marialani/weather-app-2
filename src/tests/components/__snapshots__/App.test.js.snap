// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="weather-app"
  >
    <h1
      class="location-details"
    >
      , 
    </h1>
    <div
      class="search-form"
    >
      <form>
        <input
          name="search"
          placeholder="Search city..."
          type="text"
          value=""
        />
        <button
          type="submit"
        >
          Search
        </button>
      </form>
    </div>
    <div
      class="forecast-summaries"
    />
  </div>
</DocumentFragment>
`;
